aws/rust-runtime/aws-config/src/default_provider/credentials.rs

```diff
@@ -114,6 +114,71 @@ impl Builder {
         self
     }
 
+    /// Timeout for the given [`ProvideCredentials`] implementation.
```

Comment:
should this say "default credentials chain"? We should clarify that this is the entire chain and not an individual provider (but maybe we should actually add that too...)

Reply:
Good call. Fixed.